src/main/java/com/greedobank/cards/client/UserClient.java

```diff
@@ -0,0 +1,43 @@
+package com.greedobank.cards.client;
+
+import com.greedobank.cards.model.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class UserClient {
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    @Value("${url}")
```

Comment:
we have only one URL per all the project? how can someone determine what this URL is for when reading properties?

Reply:
Done - rename it